src/foam/nanos/notification/email/DAOEmailService.js

```diff
@@ -123,11 +123,11 @@ if (foam.util.SafetyUtil.isEmpty(emailMessage.getSubject())) {
   emailMessage.setSubject(templateSubject.render(model));
 }
 
-// If sendTo isn't set in the message
-// and sendTo is provided in the template, use the sendTo from template
+/* If To isn't set in the message and sendTo is provided in the template, use
+  the sendTo from template. */
 String sendTo = emailTemplate.getSendTo();
-if ( foam.util.SafetyUtil.isEmpty(emailMessage.getTo()) &&
-  ! foam.util.SafetyUtil.isEmpty(emailTemplate.getsendTo()) ) {
+if ( emailMessage.getTo().length == 0  &&
+  ! foam.util.SafetyUtil.isEmpty(emailTemplate.getSendTo()) ) {
```

Comment:
Why this change?  Just puzzled.

Reply:
The PR for the Email was merge earlier today so guessing Dhiren's was having git/merge issues.

Reply:
The property `to` in EmailMessage.js is an array, whereas `isEmpty` assumes it is a string. This was throwing compilation errors. Hence, I had to change the logic to fix things.